newproject.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="sv" dir="ltr">
+	<head>
+		<meta charset="utf-8">
+		<title>Nytt projekt</title>
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+		<link href="https://fonts.googleapis.com/css?family=Fira+Sans:300,400" rel="stylesheet">
+		<link rel="stylesheet" href="css/newproject.css">
+	</head>
+	<body onload="getAllUsers()">
```

Comment:
This is what is reffered to as "unsafe-inline" and can often be blocked by CSP (Content Security Policy) please use an eventlistner on `document` with the trigger `DOMContentLoaded` or similar such as `document.onload` or `window.onload`